app/src/openxr/cpp/OpenXRInput.cpp

```diff
@@ -107,10 +107,10 @@ std::string OpenXRInput::GetControllerModelName(const int32_t aModelIndex) const
   return mInputSources.at(aModelIndex)->ControllerModelName();
 }
 
-void OpenXRInput::UpdateInteractionProfile()
+void OpenXRInput::UpdateInteractionProfile(ControllerDelegate& controller)
 {
   for (auto& input : mInputSources) {
-    input->UpdateInteractionProfile();
+    input->UpdateInteractionProfile(controller);
```

Comment:
We're still using controller here instead of delegate.

Reply:
This slipped too. Fixed.